src/System.Threading.Channels/tests/ChannelTestBase.netcoreapp.cs

```diff
@@ -252,7 +252,7 @@ public async Task ReadAllAsync_CanceledBeforeMoveNextAsync_Throws(bool dataAvail
             var cts = new CancellationTokenSource();
             cts.Cancel();
 
-            IAsyncEnumerator<int> e = c.Reader.ReadAllAsync().GetAsyncEnumerator(cts.Token);
+            IAsyncEnumerator<int> e = c.Reader.ReadAllAsync(cts.Token).GetAsyncEnumerator();
```

Comment:
These CT changes are only necessary because we don't yet have the compiler changes that recognize the attribute. Once we do, I'll add more tests that do it both ways.

Reply:
consider leaving a comment, so whoever updates the compiler reference isn't surprised

Reply:
Surprised? Nothing will break, right?

Reply:
ah, good point. This code will still work (`default` token in `GetAsyncEnumerator` will be ignored).